src/app/pages/storage/components/dashboard-pool/topology-card/topology-card.component.ts

```diff
@@ -20,7 +23,7 @@ interface TopologyState {
 }
 
 const notAssignedDev = 'VDEVs not assigned';
```

Comment:
Feels weird comparing to a string down below in code. Should be something else and string itself should be translated.